modules/acs-architecture-overview.adoc

```diff
@@ -15,7 +15,10 @@ However, the underlying components and the interactions between them remain the
 
 .{product-title-short} architecture for Kubernetes
 
-The following graphic shows the architecture with the StackRox Scanner. For version 4.4, Scanner V4 is available. Installation of Scanner V4 is optional, but provides additional benefits.
+The following graphic shows the architecture with the StackRox Scanner. For version 4.4, Scanner V4 (Technology Preview) is available. Installation of Scanner V4 is optional, but provides additional benefits.
```

Comment:
I haven't really seen people using "(Technology Preview)" after TP features in normal sentences like this. Only in headings, etc. I would think that the following TP note should be sufficient :shrug:

Reply:
OK, I wasn't sure how strict I needed to be about it, but I'll remove some of them because I felt they were a bit jarring too!